src/ShiftedProximalOperators.jl

```diff
@@ -156,6 +180,39 @@ the nonsmooth term is applied.
 """
 @inline prox_zero(q::R, l::R, u::R) where {R <: Real} = min(max(q, l), u)
 
+"""
+    iprox_zero(d, g, l, u)
+
+Return the solution of
+
+    min ½ d y² + g y subject to l ≤ y ≤ u
+
+for any d, g, assuming that 0 ∈ [l, u] if d < eps() and g = 0.
```

Comment:
Should it be |d| < eps()? Why those restrictions? If l and u are finite, we can always solve the problem.

Reply:
Is is clearer like this:

If `d < eps()` and `g = 0`, we assume that `0 ∈ [l, u]`. 

In fact, for our use in RegularizedOptimization, we always have `0 ∈ [l, u]` and if the solution of the prox/iprox is zero, we say that we converged and we stop the algorithm. This is why I set `y = 0` in the case you mentioned below.